### PR TITLE
feature: don't build plasma-deployer

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

File: tester/Dockerfile.plasma_deployer
Comment (unnawut) [inline tester/Dockerfile.plasma_deployer]: It turns out the failed test in https://github.com/omisego/elixir-omg/pull/1056 is due to this line of code. This is pulling `plasma-contracts` that's about 10 days old. There's been a quite a bit of changes since then.

Can this be bumped to https://github.com/omisego/plasma-contracts/commit/4fe49008625181df6a9b81f0357f849ccf413acd or even better https://github.com/omisego/plasma-contracts/commit/a6d870f4562b359bece7bea01643e4a6b567b5a9 with the `MIN_EXIT_PERIOD` support?

**Review by sirn — APPROVED** [review]
Apart from what @unnawut suggest 👌

**Comment by unnawut** [comment]
Double checked with sirn, he meant the SHA should be bumped before merge. Let's do it!

File: tester/Dockerfile.plasma_deployer
Comment (InoMurko) [reply tester/Dockerfile.plasma_deployer]:
```suggestion
RUN cd /home/node/plasma-contracts && git reset --hard 1608296ef56ab98d857321a687c10dcdd86d72d3
```

@unnawut ? wdyt?

Comment (unnawut) [reply tester/Dockerfile.plasma_deployer]: Yes lovely please <3

**Comment by unnawut** [comment]
Just thought of this… do we need the `Dockerfile.plasma_deployer` built as part of `docker-elixir-omg`?

Can we instead make a `plasma-contracts/Dockerfile` and setup a CI docker build for each plasma-contracts commit?

This way we can change 2 references in elixir-omg and be done… unless I’m missing something…

---

This question doesn't block this PR from being merged though.

**Pull request merged**
